src/lib/updating-element.ts

```diff
@@ -288,35 +270,18 @@ export abstract class UpdatingElement extends HTMLElement {
     // metadata.
     this._ensureClassProperties();
     this._classProperties!.set(name, options);
-    if (!options.noAccessor) {
-      const superDesc = descriptorFromPrototype(name, this.prototype);
-      let desc;
-      // If there is a super accessor, capture it and "super" to it
-      if (superDesc !== undefined && (superDesc.set && superDesc.get)) {
-        const {set, get} = superDesc;
-        desc = {
-          get() { return get.call(this); },
-          set(value: any) {
-            const oldValue = this[name];
-            set.call(this, value);
-            this.requestUpdate(name, oldValue);
-          },
-          configurable : true,
-          enumerable : true
-        };
-      } else {
+    if (!options.noAccessor && !this.prototype.hasOwnProperty(name)) {
         const key = typeof name === 'symbol' ? Symbol() : `__${name}`;
-        desc = {
-          get() { return this[key]; },
-          set(value: any) {
-            const oldValue = this[name];
-            this[key] = value;
+      const desc = {
```

Comment:
you can inline this into the defineProperty call now

Reply:
Done.